source/contributors/_index.md

```diff
@@ -4,100 +4,181 @@ url: /contributors/
-contributors come and go from the field.
+### Download
 
-In particular, we have some [etiquette and behavior expectations][8] when 
-you work with ASF communities.
+Most (not all) Apache projects will have a prominent download link on
+the front page of their project website. This will (hopefully) be
+accompanied by instructions on how to download, build, install, and use
+the releases.
+
+Getting to know how to use the software that a project releases is a
+reasonable first step towards contributing. Keep notes of that first
+user experience, as that is valuable information for the project, and
+might be a good place to start your contributions.
+
+### The Code
+
+Most (not all) Apache projects have their code on
+[GitHub](https://github.com/apache). Search for the project name in the
+**apache** GitHub organization.
+
+### Communication
+
+Most (not all) Apache project have their main discussions on [mailing
+lists][lists]. Most projects have
+`users@project.apache.org` and `dev@project.apache.org` mailing lists,
+focused, respectively, on end-user questions, and development activity.
+You can read the archives (i.e., messages from the past), as well as
+participate in current discussions, either by subscribing to the lists,
+or by using the [lists.apache.org][lists] site
+directly.
+
+#### Subscribe
+
+To subscribe (that is, to receive every message sent to the list via
+email) to a project mailing list (e.g., if you want to subscribe
+to the `users` list for `PROJECT`)
+send a blank email message to `users-subscribe@PROJECT.apache.org`.
+Substitute the name of the list (i.e., `dev` instead of `users`), and of 
+the specific project you're interested in (i.e., `karaf` instead of
+`PROJECT`).
+
+#### Unsubscribe
+
+To unsubscribe, the process is similar. Send a blank email message to
+`users-unsubscribe@PROJECT.apache.org` from the email address with which
+you subscribed.
+
+In each case, you'll receive a confirmation email, with further
+instructions.
+
+#### Other communication channels
+
+Some projects use other communication channels, such as Slack, Discord,
+or WiChat. This will (hopefully) be listed on the project page.
 
-See also this [detailed advice for how to become a committer](/contributors/becomingacommitter.html)
+## Your First Contribution
 
-## Anyone Can Become A Committer
+Most first contributions are simple, as you become familiar with the
+project and its specific contributor flow.
 
-There is nothing at the ASF that says you must write code 
-in order to be a committer. Anyone who is supportive by contributing towards project,
-community, documentation or code is a likely candidate for committership.
+We recommend that you start by finding, and joining, the relevant
+communication channel. Introduce yourself, and say what you're
+interested in working on. How people respond to this can be a good
+indicator of how welcoming the project is. It will also be an
+opportunity for them to point you to good first issues,
```

Comment:
we have https://community.apache.org/committers/good-first-issues.html to explain that

Reply:
Ooh. Right. Thanks.